Source/DataSources/KmlDataSource.js

```diff
@@ -1357,7 +1502,7 @@ define([
             var length = childNodes.length;
             for (var q = 0; q < length; q++) {
                 var child = childNodes[q];
-                if (child.localName === featureName && namespaces.kml.indexOf(child.namespaceURI) !== -1) {
+                if (child.localName === featureName) {
```

Comment:
I think we may have had a bug related to this, but I don't remember the details.  What issue does this change fix?